{
  "Header": {
    "Original": "CVE-2020-28912 (With MariaDB running on Windows, when local clients connect to the ser ...)",
    "Line": 138218,
    "ID": "CVE-2020-28912",
    "Description": "(With MariaDB running on Windows, when local clients connect to the ser ...)"
  },
  "Annotations": [
    {
      "Original": "- mariadb-10.5 \u003cnot-affected\u003e (Only affects MariaDB on Windows)",
      "Line": 138219,
      "Type": "package",
      "Package": "mariadb-10.5",
      "Kind": "not-affected",
      "Description": "Only affects MariaDB on Windows"
    },
    {
      "Original": "- mariadb-10.3 \u003cnot-affected\u003e (Only affects MariaDB on Windows)",
      "Line": 138220,
      "Type": "package",
      "Package": "mariadb-10.3",
      "Kind": "not-affected",
      "Description": "Only affects MariaDB on Windows"
    },
    {
      "Original": "- mariadb-10.1 \u003cnot-affected\u003e (Only affects MariaDB on Windows)",
      "Line": 138221,
      "Type": "package",
      "Package": "mariadb-10.1",
      "Kind": "not-affected",
      "Description": "Only affects MariaDB on Windows"
    },
    {
      "Original": "NOTE: https://jira.mariadb.org/browse/MDEV-24040",
      "Line": 138222,
      "Type": "NOTE",
      "Description": "https://jira.mariadb.org/browse/MDEV-24040"
    },
    {
      "Original": "NOTE: https://github.com/MariaDB/server/commit/3829b408d6",
      "Line": 138223,
      "Type": "NOTE",
      "Description": "https://github.com/MariaDB/server/commit/3829b408d6"
    }
  ]
}